src/workerfarm/WorkerFarm.js

```diff
@@ -0,0 +1,284 @@
+const {EventEmitter} = require('events');
+const os = require('os');
+const errorUtils = require('./errorUtils');
+const Worker = require('./Worker');
+
+let shared = null;
+class WorkerFarm extends EventEmitter {
+  constructor(options, farmOptions = {}) {
+    super();
+    this.options = Object.assign(
+      {
+        maxConcurrentWorkers: WorkerFarm.getNumWorkers(),
+        maxConcurrentCallsPerWorker: 10,
+        forcedKillTime: 100,
+        warmWorkers: true,
+        useLocalWorker: true,
+        workerPath: require.resolve('../worker')
```

Comment:
This will fail in Node 6 as the test environment runs using babel-register and therefore out of src instead of lib